frontend/view-edit-client/client-history/client-history.component.tsx

```diff
@@ -335,12 +342,21 @@ const allFiltersOn: ClientHistoryFilterOptions = Object.keys(LogType).reduce(
 
 function getInitialLogState(): LogState {
   const localStorageFilters = localStorage.getItem("cu:client-history-filters");
+  let filters;
+  if (localStorageFilters) {
+    filters = JSON.parse(localStorageFilters);
+    Object.keys(LogType).forEach(logType => {
+      if (!filters.hasOwnProperty(logType)) {
```

Comment:
if they have filters stored in local storage that don't include newly added logTypes, default to the newly added logTypes being on.